.github/workflows/make_leo_pr.yml

```diff
@@ -29,14 +36,36 @@ jobs:
         env:
           BROADBOT_TOKEN: ${{ secrets.BROADBOT_TOKEN }}
         run: |
+          function getVersionFromReference(){
+            cat $1 | \
+            sed -e '/cromwellApp/,/chartVersion/!d' -e 's:^[ ]*::' | \
```

Comment:
Maybe a comment explaining what this is doing, and what type of file to apply it to?